src/payment/strategies/ppsdk/ppsdk-strategy.ts

```diff
@@ -43,7 +42,21 @@ export class PPSDKStrategy implements PaymentStrategy {
         return this._store.getState();
     }
 
-    async finalize(_options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+    async finalize(options?: PaymentRequestOptions): Promise<InternalCheckoutSelectors> {
+        const { bigpayBaseUrl } = this._store.getState().config.getStoreConfigOrThrow().paymentSettings;
```

Comment:
Maybe destruction can help to combine the `getState` call into one. 🍹 i.e.:

```ts
const { config: { getStoreConfigOrThrow }, order: { getPaymentId } } = this._store.getState();
const { bigpayBaseUrl } = getStoreConfigOrThrow().paymentSettings;
const paymentId = getPaymentId(options?.methodId);
```

Reply:
:+1 one for the next refactor